### PR TITLE
Issue #37: Build imagick from repo and merge apt-get to one call

### LINKED ISSUE #37: Build imagick from repo

### PR DESCRIPTION
**Pull request by Schrank**
Fixes #37

### CODE REVIEW

File: provisioning/scripts/init-system.sh [hunk 1]
```diff
@@ -33,42 +33,14 @@ ln -sf /etc/nginx/sites-available/demo.lizardsandpumpkins.com.loc /etc/nginx/sit
...
-grep -q -F 'extension=imagick.so' /etc/php/7.0/cli/php.ini || echo extension=imagick.so >> /etc/php/7.0/cli/php.ini
+apt-get -y install php-fpm php-mysql php-curl php7.0-intl php7.0-gd php7.0-mbstring php7.0-xml php7.0-xdebug php7.0-imagick curl unzip
 
 sed -i -e 's/^display_errors = Off$/display_errors = On/' /etc/php/7.0/fpm/php.ini
 sed -i -e 's/^display_errors = Off$/display_errors = On/' /etc/php/7.0/cli/php.ini
 
 echo xdebug.remote_enable=On >> /etc/php/7.0/cli/conf.d/20-xdebug.ini
 echo xdebug.remote_connect_back=On >> /etc/php/7.0/cli/conf.d/20-xdebug.ini
 
-/etc/init.d/php7.0-fpm reload
```
Comment (tim-bezhashvyly) [inline provisioning/scripts/init-system.sh]: You sure it is not required? Some configuration manipulations happened at 4 lines above.

File: provisioning/scripts/init-system.sh [hunk 2]
```diff
@@ -33,42 +33,14 @@ ln -sf /etc/nginx/sites-available/demo.lizardsandpumpkins.com.loc /etc/nginx/sit
...
-git clone --depth 1 https://github.com/mkoppanen/imagick.git
-cd imagick/
-phpize
-./configure
-make
-make install
-
-grep -q -F 'extension=imagick.so' /etc/php/7.0/fpm/php.ini || echo extension=imagick.so >> /etc/php/7.0/fpm/php.ini
-grep -q -F 'extension=imagick.so' /etc/php/7.0/cli/php.ini || echo extension=imagick.so >> /etc/php/7.0/cli/php.ini
+apt-get -y install php-fpm php-mysql php-curl php7.0-intl php7.0-gd php7.0-mbstring php7.0-xml php7.0-xdebug php7.0-imagick curl unzip
```
Comment (tim-bezhashvyly) [inline provisioning/scripts/init-system.sh]: I believe you tested it? I mean destroying VM, building it and checking all (projection and content delivery) is working?

Comment (Schrank) [inline provisioning/scripts/init-system.sh]: Can't see any problem and 

    vagrant@demo:/var/log$ sudo tail -f php7.0-fpm.log mysql.err syslog nginx/error.log /vagrant/sample-project/share/log/*

doesn't write anything

Comment (Schrank) [inline provisioning/scripts/init-system.sh]: I assume, we only needed it for imagick (edited)

**Pull request merged**